<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryWindow</class>
 <widget class="QMainWindow" name="EntryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>430</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Eingabe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Simulation Zweikörperproblem&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Zentralkörper</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>538</width>
             <height>212</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Masse</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="central_mass"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;kg&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(Standard: 5.972*10&lt;span style=&quot; vertical-align:super;&quot;&gt;24&lt;/span&gt; kg)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Radius</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="central_radius"/>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;m&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(Standard: 6371 km)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="central_v0_x"/>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>m/s</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string>(Standard: 0 m/s)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="central_v0_y"/>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>m/s</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string>(Standard: 0 m/s)</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="central_v0_z"/>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>m/s</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>(Standard: 0 m/s)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" rowspan="3">
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Startgeschwindigkeit&lt;/p&gt;&lt;p&gt;(x, y, z)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Zweitkörper</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>538</width>
             <height>212</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Masse</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="sat_mass"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;kg&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(Standard: 500 kg)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Radius</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="sat_radius"/>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;m&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(Standard: 2 m)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="sat_v0_x"/>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>m/s</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>(Standard: 0 m/s)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="sat_v0_y"/>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>m/s</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>(Standard: 0 m/s)</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="sat_v0_z"/>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>m/s</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>(Standard: -8000 m/s)</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_16">
              <property name="toolTip">
               <string>Start-Distanz zum Zentralkörper</string>
              </property>
              <property name="text">
               <string>Start-Distanz</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="distance"/>
            </item>
            <item row="5" column="2">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;m&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(Standard: 1000 km)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" rowspan="3">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Startgeschwindigkeit&lt;/p&gt;&lt;p&gt;(x, y, z)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(Standardwerte werden angewendet wenn keine Werte angegeben werden.)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="b_reset">
        <property name="text">
         <string>Zurücksetzen</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_ok">
        <property name="text">
         <string>&amp;OK, Simulation starten</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptionen">
    <property name="title">
     <string>Optionen</string>
    </property>
    <addaction name="actionEinstellungen"/>
    <addaction name="separator"/>
    <addaction name="actionVerlassen"/>
    <addaction name="actionNeu_starten"/>
   </widget>
   <widget class="QMenu" name="menuBeispiele">
    <property name="title">
     <string>Werte</string>
    </property>
    <addaction name="actionVoreinstellungen"/>
    <addaction name="separator"/>
    <addaction name="actiongespeicherte_Werte_laden"/>
    <addaction name="actionWertedatei_oeffnen"/>
    <addaction name="separator"/>
    <addaction name="actionWerte_speichern"/>
    <addaction name="actionWertedatei_speichern_unter"/>
   </widget>
   <addaction name="menuOptionen"/>
   <addaction name="menuBeispiele"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEinstellungen">
   <property name="text">
    <string>Einstellungen...</string>
   </property>
   <property name="toolTip">
    <string>Einstellungen öffnen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionVerlassen">
   <property name="text">
    <string>Verlassen</string>
   </property>
   <property name="toolTip">
    <string>Programm schliessen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actiongespeicherte_Werte_laden">
   <property name="text">
    <string>Laden</string>
   </property>
   <property name="toolTip">
    <string>Werte laden und eintragen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionVoreinstellungen">
   <property name="text">
    <string>Voreinstellungen...</string>
   </property>
   <property name="toolTip">
    <string>Voreinstellungen öffnen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionWertedatei_oeffnen">
   <property name="text">
    <string>Öffnen von...</string>
   </property>
   <property name="toolTip">
    <string>Wertedatei öffnen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionWerte_speichern">
   <property name="text">
    <string>Speichern</string>
   </property>
   <property name="toolTip">
    <string>Werte speichern</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionWertedatei_speichern_unter">
   <property name="text">
    <string>Speichern unter...</string>
   </property>
   <property name="toolTip">
    <string>Werte speichern unter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNeu_starten">
   <property name="text">
    <string>Neu starten</string>
   </property>
   <property name="toolTip">
    <string>Programm neu starten</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>b_ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
